ColabTool/LinkPost/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { PropTypes } from 'react';
+import {
+  Card,
+  Image,
+  Link,
+  LinkifiedText,
+  Text,
+} from '@bufferapp/components';
+import style from './style.css';
+import Post from '../Shared/Post';
+
+const LinkPost = ({
+  isConfirmingDelete,
+  isDeleting,
+  isWorking,
+  imageSrc,
+  links,
+  linkAttachment,
+  manager,
+  onApproveClick,
+  onCancelConfirmClick,
+  onDeleteClick,
+  onDeleteConfirmClick,
+  onEditClick,
+  profile,
+  text,
+}) => {
+  const children = (
+    <div className={style['post-content']}>
+      <div className={style['post-content-text']}>
+        <LinkifiedText
+          links={links}
+        >
+          {text}
+        </LinkifiedText>
+      </div>
+      <div className={style['link-attachment-container']}>
+        <Link href={linkAttachment.url}>
```

Comment:
Will update this use of `Link` to include the new `unstyled` config once this PR is merged into buffer-components: https://github.com/bufferapp/buffer-components/pull/34/files